kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/network/model/OfferList.java

```diff
@@ -16,27 +15,39 @@
     @SerializedName("paging")
     private Paging paging = null;
 
-    public OfferList offers(List<Offer> offers) {
-        this.offers = offers;
-        return this;
+    public OfferList() {
+        this.offers = new ArrayList<>();
     }
 
-    public OfferList addOffersItem(Offer offersItem) {
+    public boolean addAtIndex(final int index, @NonNull final Offer offer) {
 
         if (this.offers == null) {
-            this.offers = new ArrayList<Offer>();
+            this.offers = new ArrayList<>();
+            return this.offers.add(offer);
+        } else {
+            try {
+                this.offers.add(index, offer);
+                return true;
+            } catch (IndexOutOfBoundsException e) {
+                return false;
+            }
+        }
+    }
+
+    public OfferList addAll(@NonNull final OfferList offerList) {
+        List<Offer> offers = offerList.getOffers();
+        if (offers != null) {
+            this.offers.addAll(size(), offers);
```

Comment:
if offers== null?  anywise it no need to check offers != null right as it set in constructor and never set to null?

Reply:
no need to check offers != null anymore

Reply:
yup